.github/workflows/fund_user.yml

```
@@ -0,0 +1,49 @@
+name: Fund PSN user trough L1 ERC20 Token Bridge
+
+on:
+  workflow_dispatch:
+    inputs:
+      account_address:
+        description: 'PSN address of the account to transfer to(0x23213123....)'
+        required: true
+        type: string
+      amount:
+        description: 'Amount to transfer in whole tokens(100 USDC for example)'
+        required: true
+        type: string
+
+env:
+  PYTHON_VERSION: "3.9"
+  PARACLEAR_L1_ADMIN_PRIVATE_KEY: ${{ secrets.PARACLEAR_L1_ADMIN_KEY }}
+  PARACLEAR_L1_BRIDGE_ADDRESS: "0xBC5BF8f315B9D1Eebd8062CAdf113A05c5abB43E"
+  PARACLEAR_L1_TOKEN_ADDRESS: "0x66aee3c05916df7361197c0ffed6a0fdeb0b2b09b3a9bab48ac19505b82fcd"
+  WEB3_INFURA_PROJECT_ID: ${{ secrets.PARACLEAR_L1_BRIDGE_INFURA_API_KEY }}
+  AMOUNT: ${{ github.event.inputs.amount }}
+  PARACLEAR_L2_USER_ADDRESS: ${{ github.event.inputs.l2_address }}
```

Or maybe we want to rename input `account_address` to `l2_address`?

```suggestion
  PARACLEAR_L2_USER_ADDRESS: ${{ github.event.inputs.account_address }}
```

---

we want to rename the input yes!